recipes/datatables/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "datatables" %}
+{% set version = "1.10.16" %}
+{% set sha256 = "3f4e44350bb80b6f4afa5d2841ff14fc6aabe176b6e51e3c3fe0324b040ebf73" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataTables/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
```

Comment:
Can you please fix this as in the rest of the PRs?

Reply:
done

Reply:
@isuruf : It has passed all checks. Please let me know if you have any more suggestions.